tests/test_warm_reboot.py

```diff
@@ -1975,8 +1973,8 @@ def test_system_warmreboot_neighbor_syncup(self, dvs, testlog):
         check_kernel_reachable_v4_neigh_num(dvs, NUM_OF_NEIGHS)
         check_kernel_reachable_v6_neigh_num(dvs, NUM_OF_NEIGHS)
 
-        check_kernel_stale_v4_neigh_num(dvs, NUM_OF_NEIGHS/2)
-        check_kernel_stale_v6_neigh_num(dvs, NUM_OF_NEIGHS/2)
+        #check_kernel_stale_v4_neigh_num(dvs, NUM_OF_NEIGHS/2)
```

Comment:
What's the purpose of commenting these out?

Reply:
Just remove it not needed

---
In reply to: [382734921](https://github.com/Azure/sonic-swss/pull/1197#discussion_r382734921) [](ancestors = 382734921)

Reply:
done